src/main/java/micdoodle8/mods/galacticraft/core/asm/GCTransformer.java

```diff
@@ -0,0 +1,78 @@
+package micdoodle8.mods.galacticraft.core.asm;
+
+import cofh.asm.LoadingPlugin;
+import com.google.common.collect.ImmutableMap;
+import cpw.mods.fml.relauncher.Side;
+import net.minecraft.crash.CrashReport;
+import net.minecraft.launchwrapper.IClassTransformer;
+import net.minecraft.util.ReportedException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.ClassWriter;
+import org.objectweb.asm.util.TraceClassVisitor;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import static micdoodle8.mods.galacticraft.core.asm.GCLoadingPlugin.debugOutputDir;
+import static micdoodle8.mods.galacticraft.core.asm.GCLoadingPlugin.dev;
+import static org.objectweb.asm.Opcodes.ASM5;
+
+public class GCTransformer implements IClassTransformer {
+    static final Logger log = LogManager.getLogger("TC4TweakTransformer");
```

Comment:
Is this supposed to be the same logger as TC4Tweaks?

Reply:
eh not quite.